cli/Valet/Context/Context.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Valet\Context;
+
+abstract class Context
+{
+    public static function assign()
+    {
+        if (static::isHerd()) {
+            return new Herd();
+        }
+
+        return new Standalone();
+    }
+
+    public static function isHerd()
+    {
+        return true; // @todo implement
```

Comment:
This should work in the release version: `basename($_SERVER['SCRIPT_NAME']) == 'herd.phar'`

Not sure what the solution is for non-release version.